cli/cmd/deploy/cmd.go

```diff
@@ -0,0 +1,87 @@
+package deploy
+
+import (
+	"errors"
+	"net/http"
+	"os"
+
+	"numerous/cli/internal/app"
+	"numerous/cli/internal/gql"
+
+	"github.com/spf13/cobra"
+)
+
+var DeployCmd = &cobra.Command{
+	Use:   "deploy [app directory]",
+	Run:   run,
+	Short: "Deploy an app to an organization.",
+	Long: `Deploys an application to an organization on the Numerous platform.
+
+An apps deployment is identified with the <name> and <organization> identifier.
+Deploying an app to a given <name> and <organization> combination, will override
+the existing version.
+
+The <name> must contain only lower-case alphanumeric characters and dashes.
+
+After deployment the deployed version of the app is available in the
+organization's apps page.
+
+If no [app directory] is specified, the current working directory is used.`,
+	Example: `
+If an app has been initialized in the current working directory, and it should
+be pushed to the organization "organization-slug-a3ecfh2b", and the app name
+"my-app", the following command can be used:
+
+	numerous deploy --organization "organization-slug-a3ecfh2b" --name "my-app"
+	`,
+	Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
I see you also set a variable here. This is mostly just something to consider, that we should maybe use cobras own  argument functions such as `cobra.RangeArgs(0, 1)` in this case

Reply:
Do you mean to move the logic of setting `appDir`?

I am not sure what is better practices, but to me it seemed that `cobra.RangeArgs` indicates a list of arguments, and that it is nice to keep the logic related to parsing the arguments in the `Args` function.

I changed the error message to be more specific to this command, which I think will give more useful output than `RangeArgs`.